getting_started/intel_debug/introduction.rst

```diff
@@ -142,60 +142,61 @@ wrong file is selected at any of the three layers.
 4. Chromebooks and SOF
 ----------------------
 
-As stated above, starting from 2019/2020 Intel Chromeboooks have been
-configured with the *community* key which means they can run
+As stated above, starting from 2019/2020, Intel Chromeboooks have been
+configured with the *community* key. It means that Chromebooks can run
 audio firmware signed by anyone. The entire filesystem is locked by
-default instead but there are a couple options to disable security for
-development purposes. In all cases the very first step is to switch the
+default instead, but there are several options to disable security for
+development purposes. In all cases the first step is to switch the
 Chromebook to (non-secure) `Developer Mode
 <https://chromium.googlesource.com/chromiumos/docs/+/HEAD/developer_mode.md>`_.
-Developer Mode is in fact the only strictly required step if you only
+Developer Mode is the only required step if you only
 want to install and run your own SOF firmware and are not interested in
-changing anything else in ChromeOS.
+changing anything else in Chrome OS.
 
 If you need the flexibility to make more changes, Chromebooks can run
-Linux in several, non-mutually exclusive ways. All the options listed
-below will let you run any SOF firmware at will. One of the biggest
+Linux in several non-mutually exclusive ways. All the options listed
+below let you run any SOF firmware. One of the biggest
 differences between them is how to install and run your own Linux
 kernel.
 
-- **ChromeOS** has direct hardware access but ChromeOS development
-  cannot happen on ChromeOS itself, it requires a separate workstation
-  similar to how most embedded development typically does. Setting up
-  the ``cros_sdk`` is documented in the `ChromeOS developer guide
+- **Chrome OS** has direct hardware access, but Chrome OS development
+  cannot happen on Chrome OS itself. It requires a separate workstation
+  similar to how most embedded development typically does. For
+  information about setting up the ``cros_sdk``, see the `Chromium OS
+  Developer Guide
   <https://chromium.googlesource.com/chromiumos/docs/+/HEAD/developer_guide.md>`_.
   The ``cros_sdk`` is a complete environment that lets you modify
-  anything in ChromeOS and even build an entire system image. The
-  ``cros_sdk`` requires significant disk space and there is some
-  learning effort if you're not already familiar with Gentoo's build
-  system "Portage"; especially for the Linux kernel.
+  anything in Chrome OS and even build an entire system image. The
+  ``cros_sdk`` requires significant disk space and some learning
+  effort if you are not already familiar with Portage, a package
+  management system in Gentoo, and Linux kernel build process.
```

Comment:
This is not what I tried to say. I meant: being familiar with Portage and especially with building the kernel in Portage.

Portage is also a package management for Gentoo but in Chrome OS the package management part is not really used, it's really the build system that is.

Reply:
@marc-hb, I created PR #387 for this correction.